bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php

```diff
@@ -952,8 +952,27 @@ public function changeChildrenSortByAction(Request $request)
     {
         $object = DataObject::getById($request->get('id'));
         if ($object) {
-            $object->setChildrenSortBy($request->get('sortBy'));
-            $object->setChildrenSortOrder($request->get('childrenSortOrder'));
+            $sortBy = $request->get('sortBy');
+            $sortOrder = $request->get('childrenSortOrder');
+
+            $currentSortBy = $object->getChildrenSortBy();
+            $currentSortOrder= $object->getChildrenSortOrder();
```

Comment:
unused var `$currentSortOrder`? should we pass this in `reindexBasedOnSortOrder()`?

Reply:
1) thx
2) I wouldn't take the value from the databse but the current value from the admin ui instead